app/src/main/java/de/danoeh/antennapod/fragment/ExternalPlayerFragment.java

```diff
@@ -203,9 +203,9 @@ private void updateUi(Playable media) {
                 .fitCenter()
                 .dontAnimate();
 
-        Glide.with(getActivity())
+        Glide.with(this)
```

Comment:
You already did a similar change a few weeks ago. Did you now catch all places where this happens or are there more?

Reply:
The leakings type of gilde have been found out.
And there are some other leaking types thart I have no idea how to fix them.